src/main/java/com/surrealdb/jdbc/SurrealJDBCConnection.java

```diff
@@ -21,272 +19,287 @@
 
 public class SurrealJDBCConnection implements Connection {
     @Override
-    public Statement createStatement() throws SQLException {
+    public Statement createStatement() {
```

Comment:
The exceptions I think are part of the interface, no? So should still be part of the signature.